docs/user/custom-domains.rst

```diff
@@ -2,125 +2,41 @@ Custom Domains
 ==============
 
 Custom domains allow you to serve your documentation from your own domain.
+For instance, this could be ``docs.example.com``.
 This is great for maintaining a consistent brand for your documentation and application.
 
-By default, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
+*By default*, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
 
 * ``<slug>.readthedocs.io`` for |org_brand|
 * ``<slug>.readthedocs-hosted.com`` for |com_brand|.
 
-For example if you import your project and it gets the :term:`slug` ``example-docs``, it will be served from ``https://example-docs.readthedocs.io``.
+How does it work?
+-----------------
```

Comment:
```suggestion
How do custom domains work?
--------------------------------
```

We should generally avoid using `it` in headings,

Reply:
Good catch, especially with `sphinx.ext.autosectionlabel`, it seems important.